hoot-services/pom.xml

```diff
@@ -136,7 +136,7 @@
                     parallelism, but haven't been able to get them to work...so forking instead. -->
                     <!-- parallel>methods</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads -->
-                    <forkCount>4</forkCount>
+                    <forkCount>16</forkCount>
```

Comment:
Is this change necessary?  There was a services test failure that was intermittent that @dmylov looked into and we wondered if it had to do with the forkCount exceeding the number of cpus on the build system.